documentation/configure.markdown

```diff
@@ -394,6 +406,52 @@ Then specify `puppetdb_migrator` as the
 See the [migration coordination documentation](migration_coordination)
 for a more detailed explanation of the process.
 
+#### Broadcasting commands to multiple databases
+
+PuppetDB can be configured to attempt to send each command to multiple
+databases on a best-effort basis.  There are no guarantees that any
+given command will reach all of the databases, so the process is
+inherently "lossy".  At the moment PuppetDB will consider a command to
+be successfully processed as soon as it manages to send the command to
+at least one of the databases.
+
+To enable broadcast, create one `[database-NAME]` subsection for each
+database.  For example, to attempt to send commands to a primary and
+secondary database, you could add something like this to your
+configuration:
+
+    [database]
+    # When subsections are defined, unset values in any subsection will
+    # default to the values set here.
+    subname = //host1:5432/puppetdb
+    username = <USERNAME1>
+    password = <PASSWORD1>
+    # Currently required
+    migrate = false
+
+    [database-primary]
+    # Settings intentionally omitted.
+    # Unset values default to those in [database]
+
+    [database-secondary]
+    subname = //host2:5432/puppetdb
+    username = <USERNAME2>
+    password = <PASSWORD2>
+
+When subsections are defined, the `[database]` section provides
+defaults that will apply whenever a value is not specified in a
+subsection.  These defaults will also be used for the read database
+whenever an explict [`[read-database]`]](#[read-database]--settings)
+isn't provided.
+
+PuppetDB will not migrate multiple databases.  This restriction may be
```

Comment:
Reminder to update docs here because we're allowing multi db migrations with cmd broadcast now.